chef-apply.gemspec

```diff
@@ -48,7 +48,7 @@ Gem::Specification.new do |spec|
                                      # localization gem...
   spec.add_dependency "toml-rb" # This isn't ideal because mixlib-config uses 'tomlrb'
                                 # but that library does not support a dumper
-  spec.add_dependency "train" # remote connection management over ssh, winrm
+  spec.add_dependency "train", ">= 1.4.24" # remote connection management over ssh, winrm
```

Comment:
Are there any notes we can add to the commit message about why this minimum version?

Reply:
I'm actually just going to get rid of this constraint. I did it to force us to use the same version the ChefDK does but that will get reconciled automatically when the ChefDK solves its entire dependency universe

Reply:
![find the universe that works](https://user-images.githubusercontent.com/517302/43473268-adab0876-94bd-11e8-8c18-e6dd26b1a1db.gif)